cli/commands/benchmark-web-vitals.mjs

```diff
@@ -155,15 +176,117 @@ function getParamsFromOptions( opt ) {
 	return params;
 }
 
+/**
+ * @param {string[]} metrics
+ * @return {Object<string, MetricsDefinitionEntry>} Metrics definition, keyed by metric identifier.
+ */
+function getMetricsDefinition( metrics ) {
+	/*
+	 * For now this only includes load time metrics.
+	 * In the future, additional Web Vitals like CLS, FID, and INP should be
+	 * added, however they are slightly more complex to retrieve through an
+	 * automated headless browser test.
+	 * See https://github.com/GoogleChromeLabs/wpp-research/pull/41.
+	 */
+	const availableMetricsDefinition = {
+		FCP: {
+			type: 'webVitals',
+			listen: 'onFCP',
+			global: 'webVitalsFCP',
+		},
+		LCP: {
+			type: 'webVitals',
+			listen: 'onLCP',
+			global: 'webVitalsLCP',
+		},
+		TTFB: {
+			type: 'webVitals',
+			listen: 'onTTFB',
+			global: 'webVitalsTTFB',
+		},
+		'LCP-TTFB': {
+			type: 'aggregate',
+			add: [ 'LCP' ],
+			subtract: [ 'TTFB' ],
+		},
+	};
+
+	/**
+	 * Server-Timing metrics can have any name, so for those a generic definition creator is used.
+	 * These metrics must be prefixed with "ST:", see below.
+	 *
+	 * @param {string} metric
+	 * @return {MetricsDefinitionEntry} Server-Timing metrics definition entry.
+	 */
+	const getServerTimingDefinition = ( metric ) => {
+		return {
```

Comment:
I'm getting a TypeScript error here:

![image](https://github.com/GoogleChromeLabs/wpp-research/assets/134745/8d383d80-e05d-4fdd-aa7d-e632cac8e3fa)

But I don't understand why, since these `MetricsDefinitionEntry` properties are all nullable.